extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -381,12 +381,11 @@ function createDicomWebApi(dicomWebConfig, servicesManager) {
        */
       const addRetrieveBulkData = instance => {
         const naturalized = naturalizeDataset(instance);
-
+        // Pratik
```

Comment:
Remove this line. ALso if same comments are there elsewhere, please take care of those.

Reply:
Resolve